test/_/release-dst/frontend/index.js

```diff
@@ -1 +1,64 @@
-(function(){'use strict';0,0,0;var a=function(b,c=200){var d;return function(...f){d||(d=setTimeout(()=>{b.apply(this,f),d=!1},c))}}(()=>{console.log('other mod')});0,0})();
\ No newline at end of file
+(function(){'use strict';var k=function(p){'use strict';function q(B,C){if(1===arguments.length){for(var D=B[0],E=1;E<B.length;E++)D=q(D,B[E]);return D}for(var F in C)if('class'===F){var G=B[F]||[];B[F]=(Array.isArray(G)?G:[G]).concat(C[F]||[])}else if('style'===F){var G=u(B[F]),H=u(C[F]);B[F]=G+(G&&H&&';')+H}else B[F]=C[F];return B}/**
+   * Process array, object, or string as a string of classes delimited by a space.
```

Comment:
This comment isn't stripped because it begins with `**`, this is how `babili` behaves by default. I think we're free to strip *all* comments in prod bundle, even if they're started with `**` (doc comment). You should dig [babili](https://github.com/babel/babili) to find out how to turn this on. I'll dig there too.